### ISSUE TITLE
[Feature]  Initialize JUPYTERLAB_DIR

### ISSUE THREAD

**Issue by costrouc** [jupyterlab/jupyterlab#5653]
I was packaging jupyterlab in [nixpkgs](https://github.com/NixOS/nixpkgs) and one issue I ran into was using a custom jupyterlab directory. I would manually have to copy the configuration of a previous jupyterlab to the new `JUPYTERLAB_DIR` directory. There already exists a `labextension clean` command to remove all extensions. It would be nice to have a similar `labextension init` to initialize the jupyterlab directory.

CC @saulshanabrook

**Comment by bobmayuze** [comment]
I think one thing you could do is to use docker volume on copying extensions if u've used docker before

**Comment by costrouc** [comment]
Yes there are workarounds. The easiest is `cp -r share/jupyter/lab JUPYTERLA_DIR` of the previous directory. However they are adhoc solutions. This is not an issue saying it's not possible just that it would be convenient to have if you want to have multiple jupyter directories for different extension sets.
